ceph.spec.in

```diff
@@ -323,6 +324,8 @@ Requires:	python-flask
 %if 0%{?suse_version}
 # python-Flask for powerdns
 Requires:      python-Flask
+# libopenssl for civetweb https support
+Requires:      libopenssl1_0_0
```

Comment:
That makes sense. All you're missing is that your colleague's knowledge of spec files is limited :) Thanks for pointing this out.